src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java

```diff
@@ -585,15 +591,22 @@ public void display(int delay, DrawingPanel fishPanel)
 					} else {
 						currentSpeed--;
 					}
+
+					Random random = new Random();
```

Comment:
Clean the random section up.  It's gross.

Reply:
Done with this part!

Reply:
No you're not!  Simplify your math.

Reply:
The algorithm is way better now, though.

Reply:
Can you give me a hint of how to simplify it?

Reply:
I'm talking about the `int num = ...` line specifically now.  When you're done, you'll have two numbers and one operator.

Reply:
Can I use Math.random() to replace it?

Reply:
Yes, and that would probably be better, 'cause it wouldn't great a new `Random` object every time through the loop.  I'm mostly complaining about `3 + 1 -(-3)` and `+ (-3)`, though.

Reply:
Fixed!